.github/workflows/processor-release.yml

```diff
@@ -71,23 +86,23 @@ jobs:
       - name: install protoc macos
         if: matrix.os == 'macos'
         run: brew install protobuf
-      - name: update linux x86_64 python 3.10
-        if: matrix.os == 'ubuntu' && matrix.python-version == '3.10'
+      - name: update python 3.10
+        if: matrix.python-version == '3.10'
         uses: actions/setup-python@v5
         with:
           python-version: "3.10"
-      - name: update linux x86_64 python 3.11
-        if: matrix.os == 'ubuntu' && matrix.python-version == '3.11'
+      - name: update python 3.11
+        if: matrix.python-version == '3.11'
         uses: actions/setup-python@v5
         with:
           python-version: 3.11
-      - name: update linux x86_64 python 3.12
-        if: matrix.os == 'ubuntu' && matrix.python-version == '3.12'
+      - name: update python 3.12
+        if: matrix.python-version == '3.12'
         uses: actions/setup-python@v5
         with:
           python-version: 3.12
-      - name: update linux x86_64 python 3.13
-        if: matrix.os == 'ubuntu' && matrix.python-version == '3.13'
+      - name: update python 3.13
+        if: matrix.python-version == '3.13'
         uses: actions/setup-python@v5
         with:
           python-version: 3.13
```

Comment:
Could you replace these individuals ones with a single entry using the matrix value?

```
with:
          python-version: "${{ matrix.python-version }}"
```

Reply:
Probably